frontend/src/components/layouts/AppLayout.tsx

```diff
@@ -400,7 +415,7 @@ function HealthIndicator() {
   return (
     <TooltipProvider>
       <Tooltip>
-        <TooltipTrigger>
+        <TooltipTrigger onClick={addFlyingBee}>
```

Comment:
Unfortunately this isn't such a good trigger, since on mobile the user must click to view the tooltip

Reply:
This is the idea of the easter egg to make it hard to find :D